src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -210,5 +212,17 @@ private void executeChangeSetMethod(Method changeSetMethod, Object changeLogInst
     }
   }
 
+  Object object;
+  private Object getObject() {
+    if (object == null) {
+      synchronized (this) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this dangerous instance of double-checked locking. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)